lib/index.rb

```diff
@@ -46,6 +47,12 @@ def all
   erb :customer_details
 end
 
+get '/items-details' do
+  @items_details = []
+  @errors = []
+  erb :items_details
+end
+
 get '/order-summary' do
   file_items_gateway = ItemsGateway.new
```

Comment:
Maybe call the class name FileItemGateway?

Reply:
Yes, our class name is called FileItemsGateway. This piece of code was done before getting the add_items_usecase code merged into master.